config/zenoh-kuksa-provider-config.json5

```diff
@@ -11,26 +11,23 @@
 
 {
     // Zenoh configuration
-    zenoh: {
+    zenoh_config: {
         // The Zenoh session mode the provider will use
         mode: "peer",
 
-        // Zenoh endpoints to connect to
-        // Can be used to connect to a zenoh-router without relying on scouting
-        connect: [
-             "tcp/zenoh-router:7447"
-        ],
+        listen: {
+            endpoints: [
+                "tcp/0.0.0.0:14444"
```

Comment:
More out of curiosity, why did you pick port 14444 here? Before your PR we used port 15000 more or less randomly.

Reply:
No particular reason, it simply sounded _even more random_ ;-)